packages/catalog-model/src/entity/Entity.ts

```diff
@@ -116,6 +116,7 @@ export type EntityMeta = JsonObject & {
    * This field can not be set by the user at creation time, and the server
    * will reject an attempt to do so. The field will be populated in read
    * operations.
+   * @deprecated replaced by etag.
```

Comment:
Not technically true - should just say that we do not support this field (the etag was never for the same purpose as generation)